helm/minio/templates/statefulset.yaml

```
@@ -1,6 +1,7 @@
 {{- if eq .Values.mode "distributed" }}
 {{ $poolCount := .Values.pools | int }}
 {{ $nodeCount := .Values.replicas | int }}
+{{ $calculatedReplicas := mul $poolCount $nodeCount }}
```

```suggestion
{{ $replicas := mul $poolCount $nodeCount }}
```

IT is visible that its calculated it is not necessary to be called as "calculated"